crates/light-client-prover/src/circuit.rs

```diff
@@ -99,8 +100,16 @@ pub fn run_circuit<DaV: DaVerifier, G: ZkvmGuest>(
                         let journal =
                             G::extract_raw_output(&proof).expect("DaData proofs must be valid");
                         let batch_proof_output: BatchProofCircuitOutput<DaV::Spec, [u8; 32]> =
-                            G::verify_and_extract_output(&journal, &batch_proof_method_id.into())
-                                .expect("Batch proof could not be verified");
+                            match G::verify_and_extract_output(
+                                &journal,
+                                &batch_proof_method_id.into(),
+                            ) {
+                                Ok(output) => output,
+                                Err(e) => {
+                                    println!("error: {:?}", e);
+                                    continue;
+                                }
+                            };
```

Comment:
just use ? and update guests to panic on errors

Reply:
This is not a place to error out, we should continue if the batch proof read from da is unverifiable. Was that not the case?

Reply:
ah sorry i thought it was lcpp

Reply:
remove println